ddp-workspace/projects/ddp-sdk/src/lib/pipes/invitationFormatter.pipe.spec.ts

```diff
@@ -18,4 +18,8 @@ describe('InvitationPipe test', () => {
     it('should format invitation code using custom chunk size and custom separator', () => {
         expect(pipe.transform('111111111111', 6, '~')).toBe('111111~111111');
     });
+
+    it('should not transform invitation code of oldv format', () => {
```

Comment:
Just wondering what is `oldv` format ?

Reply:
obviously it's a typo, should be old, thanks for noticing